src/wallet/types.ts

```diff
@@ -4,3 +4,9 @@
 export type ConnectAccountReact = {
   children?: React.ReactNode; // Children can be utilized to display customized content when the wallet is connected.
 };
+
+export type isBaseOptions = {
+  chainId: number;
+};
+
+export type isBaseResponse = { isValid: true } | { isValid: false; error: string; code: string };
```

Comment:
Do we think the wording `isValid` is right here? 🤔

Also, I wonder if we are over thinking this one, and should just be return a boolean, that's it.

Reply:
Well this is my docs example with the current working. I think it's pretty clean! 

```
import { isBase } from '@coinbase/onchainkit/wallet';

const isBaseResponse = await isBase({ chainId: '8453' });

if isBaseResponse.isValid {
  console.log('Chain id is a valid base or base sepolia chain id');
}
```

Reply:
Yeah I think for this one we should be even simpler. Let's just return a `boolean`.